### ISSUE TITLE
Test failures due to lack of gfloat formats import

### ISSUE THREAD

**Issue by kitterma** [scott-griffiths/bitstring#328]
When running the 4.2.1 test suite with gfloat 0.1, I get multiple errors:

`AttributeError: module 'gfloat' has no attribute 'formats'`

Inspecting src/gfloat/__init__.py, it says:

```
# Don't automatically import from .formats.
# If the user wants them in their namespace, they can explicitly import
# from gfloat.formats import *
```

Adding this to tests/test_fp8.py does solve the problem.  I checked and this comment is still present in their current git, so it doesn't seem to be a temporary decision.

**Comment by scott-griffiths** [comment]
Thanks - this has addressed on the main branch already as it broke my build shortly after 4.2.1 was released (see 238770c)

I'm planning a point release at the weekend that should fix it for everyone else.

**Comment by scott-griffiths** [comment]
bitstring 4.2.2 is now released. 🎉
